library/globals.inc.php

```diff
@@ -175,9 +175,9 @@ function gblTimeZones()
             xl('Pick a general theme (need to logout/login after change this setting).')
         ),
 
-        'font-family' => [
```

Comment:
@robertdown ,
Looks like @ophthal is having issues with this (using brackets instead of array() ). Do you know which PHP version is required for this?
-brady